dashboard/lib/school_info_interstitial_helper.rb

```diff
@@ -24,6 +24,15 @@ def self.show_school_info_interstitial?(user)
     return true
   end
 
+  # Determine if a user has seen the school info interstitial.
+  def self.check_last_seen_school_info_interstitial?(user)
+    return true if user.last_seen_school_info_interstitial.nil?
+
+    return true if user.last_seen_school_info_interstitial >= 7
```

Comment:
is this supposed to be 
```suggestion
    return true if user.last_seen_school_info_interstitial < 7.days.ago
```

Reply:
It should return true so the school confirmation is shown when the user signs in after or on the 7th day since the school info interstitial was seen.